datastruc_ex7.py

```diff
@@ -0,0 +1,79 @@
+import random as rnd
+
+# ex7
+# a
+
+
+def func(n: int, random: bool, lst: list):
```

Comment:
func? no documentation? you are sleeping on the couch today

Reply:
done.